core/src/main/java/org/apache/accumulo/core/spi/crypto/CryptoEnvironment.java

```diff
@@ -28,10 +32,18 @@ public interface CryptoEnvironment {
    * Where in Accumulo the on-disk file encryption takes place.
    */
   enum Scope {
-    WAL, RFILE
+    WAL, TABLE
```

Comment:
I was looking at these changes again and wondering if we should add another scope: `RECOVERY`. With my changes to the recovery code, we now have the ability to encrypt the R-files during recovery...

Reply:
Another type probably makes sense. These files use the same file type as per-table files, but contain data for multiple tables, like WAL files.